### DISCUSSION TITLE
Migration .NET 8

### DISCUSSION THREAD

**Discussion by k-placer** [e-netze/webgis-community#143] [General]
Mit Jahresende 2024 endet der Support von .NET 6, siehe  [.NET core lifecycle,](https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#lifecycle) **End of support: November 12, 2024**. 

Ein Upgrade auf .NET 8 wird damit notwendig. Wie sieht der Zeitplan für die Migration von webgis5 aus?

**Comment by gis-eni** [comment]
Das stimmt, bis eine 2024 ist zum Glück noch ein wenig Zeit.

Unser Plan ist es, dass wir immer auf den LTS Versionen (Versionen mit geraden Zahlen) bleiben und auch zeitnah Umstellen. Die Umstellung ist in der Regel für uns keine großes Thema. Wir verwenden .NET 8 bereits schon länger in anderen Projekten.

Bei WebGIS haben wir uns da bisher immer ein wenig mehr Zeit genommen, weil da erst einmal von euch die entsprechenden Runtimes installiert werden müssen. Das sollte aber mittlerweile auch keine sehr großes Thema mehr sein, weil die Runtimes ohne Probleme parallel installiert werden können. Dazu vielleicht am Mittwoch mehr.
